hypha/apply/funds/management/commands/drafts_cleanup.py

```diff
@@ -0,0 +1,80 @@
+import argparse
+from datetime import timedelta
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.utils import timezone
+
+from hypha.apply.funds.models.submissions import ApplicationSubmission
+from hypha.apply.funds.workflow import DRAFT_STATE
+
+
+def check_not_negative(value):
+    """Used to validate `older_than_days` argument
+
+    Needs to be a non-negative int
+    """
+    try:
+        ivalue = int(value)
+    except ValueError:
+        ivalue = -1
+
+    if ivalue < 0:
+        raise argparse.ArgumentTypeError(
+            f'"{value}" is an invalid non-negative integer value'
+        )
+    return ivalue
+
+
+class Command(BaseCommand):
+    help = "Delete all drafts that are older than 2 years"
```

Comment:
Works perfectly, very neat! Just update this help text and we can merge it in.

Reply:
Maybe we should add a section in the docs about neat cron commands that can be used? We have this one and the account cleanup at least.

Reply:
great catch! updated the help text & wrote a new configuration page in the docs for cron jobs. let me know if you think I missed anything